.azure/roles-only-acceptance-pipeline.yaml

```diff
@@ -0,0 +1,21 @@
+# Triggers
+# This pipeline will be triggered manually for a release or by github comment
+trigger: none
+pr:
+  autoCancel: false
+  branches:
+    include:
+      - '*'
+
+jobs:
+  - template: 'templates/system_test_general.yaml'
+    parameters:
+      name: 'acceptance_roles_only'
+      display_name: 'acceptance-roles-only'
+      test_case: '.*ST'
+      groups: 'acceptance'
```

Comment:
We should also include here the tests which are specific for this feature which means add `acceptance` tag to `RolesOnlyOperatorST` or create a specific tag for this class and change groups in yaml like `groups: 'acceptance,rolesOnly'`.  Guess a little bit better solution is to extend acceptance tests :)